WindSpigot-Server/src/main/java/org/bukkit/craftbukkit/util/Versioning.java

```diff
@@ -10,22 +10,6 @@
 
 public final class Versioning {
 	public static String getBukkitVersion() {
-		String result = "Unknown-Version";
-
-		InputStream stream = Bukkit.class.getClassLoader()
-				.getResourceAsStream("META-INF/maven/dev.cobblesword.nachospigot/api/pom.properties");
-		Properties properties = new Properties();
-
-		if (stream != null) {
-			try {
-				properties.load(stream);
-
-				result = properties.getProperty("version");
-			} catch (IOException ex) {
-				Logger.getLogger(Versioning.class.getName()).log(Level.SEVERE, "Could not get Bukkit version!", ex);
-			}
-		}
-
-		return result;
+		return "1.8.8-R0.2-SNAPSHOT";
```

Comment:
```suggestion
		return "1.8.8-R0.1-SNAPSHOT";
```
and adjust the poms to that version string
here's the discussion https://github.com/CobbleSword/NachoSpigot/pull/384